bgc_data/phytoinfo.dataresource.yaml

```diff
@@ -20,14 +20,14 @@ schema:
     type: string
     required: true
     unique: true
-  - name: FUNCTIONALGROUP
-    title: ""
-    type: string
-    required: false    
   - name: BV_CELL
     title: ""
     type: number
     required: false
+  - name: FUNCTIONALGROUP
```

Comment:
These two columns were just in the wrong order. Now they match what's on Geoserver.